lib/tokens.ts

```diff
@@ -155,17 +155,14 @@ async function getMetaplexMetadataForToken(
       const decodedMetadata = deserializeMetadata(
         tokenMetaDataAccount as unknown as RpcAccount
       );
-      const uriRes = await fetch(decodedMetadata.uri);
```

Comment:
This is actually needed as the URI contains parts of the data not just the token metadata. Its actually the URI that contains the image url. 
<img width="780" alt="Screenshot 2024-06-07 at 9 34 43 PM" src="https://github.com/metaDAOproject/futarchy-sdk/assets/1007667/e56007f0-5516-4f0d-8740-fe71c96b295d">

<img width="690" alt="Screenshot 2024-06-07 at 9 34 58 PM" src="https://github.com/metaDAOproject/futarchy-sdk/assets/1007667/1dc1525c-5471-4b30-8704-fdf265849974">


<img width="870" alt="Screenshot 2024-06-07 at 9 35 03 PM" src="https://github.com/metaDAOproject/futarchy-sdk/assets/1007667/61f74700-f12a-4f40-b897-309435779aeb">

Reply:
And ONLY for the image is why we do it this way..

Reply:
the uri is actually only the image link, so we were trying to parse the png format as json (what's inside the image url)

Reply:
<img width="923" alt="image" src="https://github.com/metaDAOproject/futarchy-sdk/assets/44983729/aa2f65a8-836b-4d90-a04f-6842276ffa82">

Reply:
https://solscan.io/token/CHqrdqTghpAyGMskfBiAyabnapEoXKk71dxJNDpmeKPB#metadata you can see here uri is the right url

Reply:
what's the standard and should we change our way to handle metadata to align?

Reply:
The metadata there isn't standard... So it's not following it, yes I'd encourage our adherence to the metadata standard.